src/main/java/com/amazon/ion/system/IonTextWriterBuilder_1_1.java

```diff
@@ -0,0 +1,367 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.ion.system;
+
+import com.amazon.ion.IonWriter;
+import com.amazon.ion.Timestamp;
+
+import java.nio.charset.Charset;
+
+/**
+ * The builder for creating {@link IonWriter}s emitting the 1.1 version of the Ion text format.
+ * <p>
+ * Builders may be configured once and reused to construct multiple
+ * objects.
+ * <p>
+ * <b>Instances of this class are not not safe for use by multiple threads
+ * unless they are {@linkplain #immutable() immutable}.</b>
+ *
+ */
+public interface IonTextWriterBuilder_1_1 extends IonWriterBuilder_1_1<IonTextWriterBuilder_1_1> {
```

Comment:
Most of the methods in here are copied from `IonTextWriterBuilder`. We duplicate them here because 1.1 users are expected to interact with the `IonTextWriterBuilder_1_1`, not the `IonTextWriterBuilder` abstract class.